tests/console/year_2038_detection.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Monitor Year 2038 problem, but so far no fix is required due to
+# https://bugzilla.suse.com/show_bug.cgi?id=1188626
+#
+# The Year 2038 (Y2038) problem relates to representing time in many digital systems
+# as the number of seconds passed since 00:00:00 UTC on 1 January 1970 and storing it
+# as a signed 32-bit integer. Such implementations cannot encode times after
+# 03:14:07 UTC on 19 January 2038. Similar to the Y2K problem, the Year 2038 problem
+# is caused by insufficient capacity used to represent time.
+#
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+use transactional qw(trup_call check_reboot_changes);
+use version_utils qw(is_transactional);
+
+sub install_pkg {
+    if (is_transactional) {
+        trup_call('pkg install chrony');
+        check_reboot_changes;
+    }
+    else {
+        zypper_call('in chrony');
+    }
+}
+
+sub run {
+    select_serial_terminal;
+
+    install_pkg if (script_run('rpm -qi chrony') != 0);
+    systemctl('start chronyd.service');
```

Comment:
Wouldn't a `chronyc makestep` be better here to ensure the systemtime is correct, regardless of how off it is? (Looking at RPi workers and others)